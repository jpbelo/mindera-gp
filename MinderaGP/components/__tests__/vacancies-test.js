import React from 'react';
import Vacancies from '../vacancies';

test('renders correctly', () => {
  const tree = renderer.create(<Vacancies />).toJSON();
  expect(tree).toMatchSnapshot();
});
