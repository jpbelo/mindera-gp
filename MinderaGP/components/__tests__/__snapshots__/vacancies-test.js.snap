// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 1334,
    }
  }
  virtual={undefined}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../components/img/logo.png",
        }
      }
      style={
        Object {
          "height": 120,
          "marginBottom": 60,
          "width": 96,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Vacancies
    </Text>
  </View>
</View>
`;
